run.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 # . /environment
+
+# Setup AnalysisBase environment in the event "$@" doesn't create a new shell
+. /release_setup.sh
+
 echo $PATH
 echo "========= all set up. ============"
 ls
-"$@"
\ No newline at end of file
+"$@"
```

Comment:
@ivukotic What is the actual command that the k8 pod executs to run the container? That would be helpful to know to understand if the example that I gave in PR body makes sense with how things should get used.